src/SFA.DAS.QnA.Application/Queries/Sequences/GetSequence/GetSequenceHandler.cs

```diff
@@ -20,10 +20,10 @@ public GetSequenceHandler(QnaDataContext dataContext, IMapper mapper)
         }
         public async Task<HandlerResponse<Sequence>> Handle(GetSequenceRequest request, CancellationToken cancellationToken)
         {
-            var application = await _dataContext.Applications.FirstOrDefaultAsync(app => app.Id == request.ApplicationId, cancellationToken: cancellationToken);
+            var application = await _dataContext.Applications.AsNoTracking().FirstOrDefaultAsync(app => app.Id == request.ApplicationId, cancellationToken: cancellationToken);
             if (application is null) return new HandlerResponse<Sequence>(false, "Application does not exist");
 
-            var sequence = await _dataContext.ApplicationSequences.FirstOrDefaultAsync(seq => seq.Id == request.SequenceId, cancellationToken: cancellationToken);
+            var sequence = await _dataContext.ApplicationSequences.AsNoTracking().FirstOrDefaultAsync(seq => seq.Id == request.SequenceId, cancellationToken: cancellationToken);
             if (sequence is null) return new HandlerResponse<Sequence>(false, "Sequence does not exist");
```

Comment:
not for this story, but we could defintely have a single endpoint that uses suitable parameters and returns the response words for 'Application does not exist' or 'sequence does not exist' I reckon